src/Microsoft.TestPlatform.CommunicationUtilities/JsonDataSerializer.cs

```diff
@@ -36,13 +41,29 @@ private JsonDataSerializer()
             DateParseHandling = DateParseHandling.DateTimeOffset,
             DateTimeZoneHandling = DateTimeZoneHandling.Utc,
             TypeNameHandling = TypeNameHandling.None,
-            ReferenceLoopHandling = ReferenceLoopHandling.Ignore
+            ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
```

Comment:
```suggestion
            ReferenceLoopHandling = ReferenceLoopHandling.Ignore
```

Reply:
It's pretty convenient to keep the last comma so that when you add a new entry you don't have to bother adding it. But I am fine if we remove it too.

Reply:
I am keeping it :)

Reply:
It's ok to me(my 2c, never liked it...looks like a typo and it's also at the end of a list so anyway you need to add another element and a comma to keep same idea so I don't see a great benefit, looks weird for younger that comes from standard programming books.)